src/documentation/domains/project/controller/project.controller.docs.ts

```diff
@@ -89,4 +89,40 @@
  *           $ref: '#/components/responses/UnauthorizedException'
  *         '404':
  *           $ref: '#/components/responses/NotFoundException'
+ *
+ *   /api/projects/user/{userId}:
```

Comment:
You are getting data from 'Projects' resource, and filtering BY user. The cognito id is in the token, so you should use that for filter the view without having to provide a query param with the user id. Regarding the name filter, that is fine.
The endpoint's name should be /api/projects. If you use it with query params: /api/projects?name=tricker. Remember cognito user id is in the token, so you don't have to provide a query param for that.

Reply:
done